app/services/remove_section_service.rb

```diff
@@ -63,10 +60,12 @@ def change_note_params
     }
   end
 
-  def notify_listeners
-    listeners.each do |listener|
-      listener.call(document, manual)
-    end
+  def discard_section
+    PublishingApiDraftSectionDiscarder.new.call(document, manual)
+  end
+
+  def export_manual
+    PublishingApiDraftManualExporter.new.call(document, manual)
```

Comment:
Minor: Keep the order of method definitions consistent, i.e. `export_manual` first.